tests/unit/DefineCompany.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { shallowWrapperFactory } from '../jest-wrapper-factory'
-import DefineCompany from '@/views/DefineCompany.vue'
+import DefineCompany from '@/views/Incorporations/DefineCompany.vue'
```

Comment:
You might be able to just import from the index file, eg,
`import { DefineCompany } from '@/views'`

Reply:
Will try!

Reply:
This works! Updated